examples/trace-analytics-sample-app/data-prepper-wait-for-os-and-start.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Please rename this file such that `os` is `opensearch`. I think `os` is overloaded.